aten/src/ATen/native/cpu/Loops.h

```diff
@@ -261,8 +308,38 @@ static inline void vectorized_outer_reduction(char** data, int64_t inner_stride,
   });
 }
 
+template <typename func_t>
+void fill_kernel(TensorIterator& iter, func_t op) {
```

Comment:
Why do we call this `fill` / `fill_kernel` instead of `nullary` / `nullary_kernel` ?

Reply:
I can change it, don't really care.